src/express/common/remote-worker.js

```diff
@@ -150,3 +150,22 @@ export async function runScpCommand(src, dest, retries) {
     }
   }
 }
+
+export async function runCommand(fn, ip, number, retries) {
+  if (retries < 0) {
+    console.log('❌ runCommand called with negative retries number: ', retries)
+    process.exit(1)
+  }
+  try {
+    const response = await fn(ip, number)
+    return response
+  } catch (error) {
+    if (retries - 1 > 0) {
+      console.log('--- running command')
```

Comment:
Shall we remove this console log, just to be consistent with other similar functions?

Reply:
Sorry missed this. Will remove.